bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -154,23 +95,23 @@ public static String getNativeReportPath() {
      */
     @NonNull
     public static String[] getCpuAbi() {
-        return getClient().deviceData.cpuAbi;
+        return getClient().getDeviceData().getCpuAbi();
     }
 
     /**
      * Retrieves global metadata from the static Client instance as a Map
      */
     @NonNull
     public static Map<String, Object> getMetadata() {
-        return new HashMap<>(getClient().metadataState.getMetadata().toMap());
+        return new HashMap<>(getClient().getMetadataState().getMetadata().toMap());
     }
 
     /**
-     * Retrieves breadcrumbs from the static Client instance as a Map
+     * Retrieves breadcrumbState from the static Client instance as a Map
      */
     @NonNull
     public static List<Breadcrumb> getBreadcrumbs() {
-        Queue<Breadcrumb> store = getClient().breadcrumbState.getStore();
+        Queue<Breadcrumb> store = getClient().getBreadcrumbState().getStore();
```

Comment:
Can this call `Client.getBreadcrumbs` rather than require a separate method to retrieve `BreadcrumbState`?

Reply:
Yep, I've updated this to use the existing method instead.